README.adoc

```diff
@@ -35,6 +36,33 @@ NOTE: If you want to use reveal.js 2, please switch your asciidoctor-reveal.js r
 
 TIP: If you are using https://pages.github.com/[GitHub Pages], plan ahead by keeping your source files on `master` branch and all output files on the `gh-pages` branch.
 
+[[node-package]]
+== Node package
+
+Using npm:
+
+ $ npm i -g npm
```

Comment:
Disclaimer: I'm a node/javascript n00b.

Why the global install of npm (with npm!?)? On my arch linux setup npm is managed by my package manager and I don't want to change that.